site/en/docs/extensions/reference/action/index.md

```diff
@@ -248,19 +245,19 @@ chrome.action.onClicked.addListener((tab) => {
 alert('Hello, world!');
 ```
 
-### Emulating pageActions with declarativeContent
+### Emulate pageActions with declarativeContent
 
-The chrome.action API replaced the [browserAction][api-browser-action] and
-[pageAction][api-page-action] APIs in Manifest V3. By default, actions are similar to browser
-actions, but it is possible to emulate the behavior of a page action using the action API.
+The `chrome.action` API replaced the [`browserAction`][api-browser-action] and
+[`pageAction`][api-page-action] APIs in Manifest V3. By default, actions are similar to Manifest V2's browser
+actions, but you can use the action API to emulate the behavior of page actions. <!--Do we want this to reference the old version? I can clean that up if needed.-->
```

Comment:
Please do.

Reply:
To clarify: what I want to do is remove the reference to how Manifest V2 works. Is that correct?